### PR TITLE
[Feature for #651] Added Contents button in toolbar for Ebooks

### PR DESCRIPTION
**Pull request by dhruvmisra**
I have added a 'Contents' button in the Ebook Reader toolbar. It uses the epub.js library to scan the XML for the contents section. If the section is found, it navigates to that section otherwise it navigates to the beginning of the book.

ebookSol

**Known bugs**
In some cases, navigating to the beginning even though contents section is present.
Cause: 'Contents' not listed in the XML of epub file.
Would love to hear your solutions.

### CODE REVIEW

**Comment by dhruvmisra** [comment]
@llaske Please review.

File: activities/EbookReader.activity/js/activity.js [hunk 1]
```diff
@@ -117,6 +117,11 @@ var app = new Vue({
 			}
 		},
 
+		showContents: function() {
+			// console.log(this.currentEpub.contents, this.$refs.view);
```
Comment (llaske) [inline activities/EbookReader.activity/js/activity.js]: Remove debug message

File: activities/EbookReader.activity/js/toolbar.js [hunk 2]
```diff
@@ -22,6 +22,7 @@ var Toolbar = {
 			<toolbar-item id="activity-button" v-bind:title="l10n.stringEbookReaderActivity"></toolbar-item>
 			<toolbar-item isSplitbar="true"></toolbar-item>
 
+			<toolbar-item ref="switchbutton" v-on:clicked="getApp().showContents()" class="toolbutton" id="contents-button" v-bind:title="l10n.stringContents" v-if="getApp().currentView == getReader()"></toolbar-item>
```
Comment (llaske) [inline activities/EbookReader.activity/js/toolbar.js]: Not the better place to add it. Will be better after the next button.
Warning: `ref` property should be unique

File: activities/EbookReader.activity/icons/lines.svg [hunk 3]
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (llaske) [inline activities/EbookReader.activity/icons/lines.svg]: Not a very intuitive icon

File: activities/EbookReader.activity/js/activity.js (hunk 1 above)
Comment (dhruvmisra) [reply activities/EbookReader.activity/js/activity.js]: Fixed now.

File: activities/EbookReader.activity/js/toolbar.js (hunk 2 above)
Comment (dhruvmisra) [reply activities/EbookReader.activity/js/toolbar.js]: Fixed.

File: activities/EbookReader.activity/icons/lines.svg (hunk 3 above)
Comment (dhruvmisra) [reply activities/EbookReader.activity/icons/lines.svg]: I have created a new icon. Please review.

**Comment by dhruvmisra** [comment]
Fixed the console error and made the required changes.
Regarding the issue, there is no direct recommendation provided in the epub.js docs. I found this implementation in one of the examples in the repository and refined it to work with our Reader.

**Comment by llaske** [comment]
Nice. Thanks.

**Pull request merged**